src/actions/account/index.ts

```diff
@@ -111,7 +105,7 @@ export const saveClaim: ThunkAction = async (
   const verifiableCredential = await identityWallet.create.signedCredential(
     {
       metadata: getClaimMetadataByCredentialType(claimsItem.credentialType),
-      claim: claimsItem.claimData,
+      claim: { ...claimsItem.claimData },
```

Comment:
This is to copy the object, isn't it? Maybe a comment would be useful to understand this change even in a few month

Reply:
Added a comment in the latest commit.